### PR TITLE
[Rev 1 Docs] [VnV Plan] Add Unit Tests

### LINKED ISSUE #554: [Rev 1 Docs] [VnV Plan] Functional Requirements Unit Tests - Users Module
**Report:** Verification and Validation Plan
**Section Number:** 6.2

### Rubric Sections
**VnV Report Rubric**
- VnV Report Matches VnV Plan

**Final Doc Rubric**
- Update VnV Plan and Report

### LINKED ISSUE #555: [Rev 1 Docs] [VnV Plan] Functional Requirements Unit Tests - Quotes Module
**Report:** Verification and Validation Plan
**Section Number:** 6.2

### Rubric Sections
**VnV Report Rubric**
- VnV Report Matches VnV Plan

**Final Doc Rubric**
- Update VnV Plan and Report

### LINKED ISSUE #556: [Rev 1 Docs] [VnV Plan] Functional Requirements Unit Tests - Appointments Module
**Report:** Verification and Validation Plan
**Section Number:** 6.2

### Rubric Sections
**VnV Report Rubric**
- VnV Report Matches VnV Plan

**Final Doc Rubric**
- Update VnV Plan and Report

### LINKED ISSUE #557: [Rev 1 Docs] [VnV Plan] Functional Requirements Unit Tests - Work Orders Module
**Report:** Verification and Validation Plan
**Section Number:** 6.2

### Rubric Sections
**VnV Report Rubric**
- VnV Report Matches VnV Plan

**Final Doc Rubric**
- Update VnV Plan and Report

### LINKED ISSUE #559: [Rev 1 Docs] [VnV Plan] Functional Requirements Unit Tests - Services Module
**Report:** Verification and Validation Plan
**Section Number:** 6.2

### Rubric Sections
**VnV Report Rubric**
- VnV Report Matches VnV Plan

**Final Doc Rubric**
- Update VnV Plan and Report

### PR DESCRIPTION
**Pull request by arkinmodi**
Close https://github.com/arkinmodi/project-sayyara/issues/554
Close https://github.com/arkinmodi/project-sayyara/issues/555
Close https://github.com/arkinmodi/project-sayyara/issues/556
Close https://github.com/arkinmodi/project-sayyara/issues/557
Close https://github.com/arkinmodi/project-sayyara/issues/559

### CODE REVIEW

**Comment by TimChoy** [comment]
The comments by Dr. Smith under the main title should also be commented out

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -1739,62 +1744,994 @@ \section{Unit Test Description} \label{Unit Test Description}
...
 \begin{enumerate}
 
-	\item{test-id1\\}
+	\item \textbf{FRT-M3-1}
 
-	Type: \wss{Functional, Dynamic, Manual, Automatic, Static etc. Most will be automatic}
+	      Type: Functional, Dynamic, Unit, Automated
 
-	Initial State:
+	      Initial State:
```
Comment (TimChoy) [inline docs/VnVPlan/VnVPlan.tex]: Should there be a N/A here if initial state is not needed or null?

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -1732,69 +1737,996 @@ \subsubsection{Legal Requirements}
...
+
+	      Type: Functional, Dynamic, Unit, Automated
+
+	      Initial State: A user exists
+
+	      Input: Authorize a user with a valid email and password
+
+	      Output: The user's data
+
+	      Test Case Derivation: A user wants to authorize themselves to view their data (e.g., appointments,
```
Comment (joyxiao99) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Test Case Derivation: A user wants to authorize themselves in order to view their data (e.g., appointments,
```

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -1732,69 +1737,996 @@ \subsubsection{Legal Requirements}
...
+
+	      Type: Functional, Dynamic, Unit, Automated
+
+	      Initial State: A user exists
+
+	      Input: Authorize a user with an invalid email and password
+
+	      Output: A unauthorized error message
+
+	      Test Case Derivation: A user wants to authorize themselves to view their data (e.g., appointments,
```
Comment (joyxiao99) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Test Case Derivation: A user wants to authorize themselves in order to view their data (e.g., appointments,
```

File: docs/VnVPlan/VnVPlan.tex [hunk 4]
```diff
@@ -1732,69 +1737,996 @@ \subsubsection{Legal Requirements}
...
+\href{https://github.com/arkinmodi/project-sayyara/blob/main/docs/Design/MIS/MIS.pdf}{MIS}. These
+tests were chosen based on common user flows.
+
+\begin{enumerate}
+
+	\item \textbf{FRT-M4-1}
+
+	      Type: Functional, Dynamic, Unit, Automated
+
+	      Initial State: A customer, a vehicle, a shop, a shop owner, and a custom service exist
```
Comment (joyxiao99) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Initial State: A customer, a vehicle, a shop, a shop owner, and a ``CUSTOM'' service exists
```

File: docs/VnVPlan/VnVPlan.tex [hunk 5]
```diff
@@ -1732,69 +1737,996 @@ \subsubsection{Legal Requirements}
...
+
+	      Test Case Derivation: A customer wants to request a quote from a shop
+
+	      How test will be performed: Jest will send a create quote request with valid information
+
+	\item \textbf{FRT-M4-2}
+
+	      Type: Functional, Dynamic, Unit, Automated
+
+	      Initial State: A customer, a vehicle, a shop, a shop owner, and a custom service exist
```
Comment (joyxiao99) [inline docs/VnVPlan/VnVPlan.tex]:
```suggestion
	      Initial State: A customer, a vehicle, a shop, a shop owner, and a ``CUSTOM'' service exists
```

**Review by joyxiao99 — APPROVED** [review]

**Pull request merged**